www/content/workstation/upgrade_labs.md

```diff
@@ -0,0 +1,537 @@
+  - Bootstrapped to the older Chef Infra Server
+  - `sudo` permissions on the node
+  - SSH connectivity between the nodes and the administrator/developer workstation
+- An administrator/developer workstation
+  - 64-bit architecture
+  - Minimum 4 GB RAM
+  - Recommended 10 GB of available disk space for installing Chef Workstation and running the Chef Upgrade Lab
+
+### Software Requirements
+
+- Meet the [platform and system requirements](https://docs.chef.io/workstation/install_workstation/) for Chef Workstation
+- Install or upgrade to the Chef Workstation [latest version](https://downloads.chef.io/chef-workstation)
+
+Chef does not prescribe any specific editor. However, the [Chef Infra extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef) for [Visual Studio Code](https://code.visualstudio.com/) features several code generators and helpful features, such as running Cookstyle each time you save a recipe.
+
+### Server Backup
+
+We recommend performing a backup before starting any server upgrade process.
+Follow the [Chef Infra Server Backup](https://docs.chef.io/runbook/server_backup_restore/) documentation before starting your Upgrade Lab.
+
+### Infrastructure Requirements
+
+#### Credentials
+
+Your credentials are set up using [knife profiles](https://docs.chef.io/workstation/knife_setup/#knife-profiles). This allows you to keep your keys in a `credentials` file, and makes switching between credentials easier.
+
+For example, in `.chef/credentials`:
+
+```toml
+[old-server]
+client_name = "user_name"
+chef_server_url = "https://old-chef-server.dev/organizations/my-org"
+client_key = """
+-----BEGIN RSA PRIVATE KEY-----
+MMM+some+key+goes+here+MMM
+-----END RSA PRIVATE KEY-----
+"""
+[new-server]
+client_name = "user_name"
+chef_server_url = "https://new-chef-server.dev/organizations/my-org"
+client_key = """
+-----BEGIN RSA PRIVATE KEY-----
+MMM+another+key+goes+here+MMM
+-----END RSA PRIVATE KEY-----
+"""
+```
+
+#### Connectivity
+
+You must have a user key for both of the Chef Infra Servers and you must be able
+to connect to both Chef Infra Servers from your development workstation.
+
+Verify connectivity by running a knife command and receiving a reasonable
+response from each server:
+
+```shell
+chef exec knife user list --profile old-server
+user_name
+chef exec knife user list --profile new-server
+user_name
+```
+
+#### Convergence
+
+Your nodes are in a good working order. They converge cleanly under the older version of Chef Infra Client.
+
+Verify that the nodes are healthy by running:
+
+```shell
+chef exec knife status --profile old-server
+```
+
+Which outputs something similar to:
+
+```output
+42 minutes ago, node-01, ubuntu 18.04.
+```
+
+This command outputs the time of the last successful chef run of each node; nodes that return radically different times for the last successful chef run are not in good working order.
```

Comment:
Is this "chef run" supposed to be a "Chef Infra Client run"?

Reply:
ok